x-pack/test/security_solution_endpoint/apps/integrations/endpoint_exceptions.ts

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { unzip } from 'zlib';
+import { promisify } from 'util';
+import expect from '@kbn/expect';
+import { IndexedHostsAndAlertsResponse } from '@kbn/security-solution-plugin/common/endpoint/index_data';
+import { EXCEPTION_LIST_ITEM_URL } from '@kbn/securitysolution-list-constants';
+import { ManifestConstants } from '@kbn/security-solution-plugin/server/endpoint/lib/artifacts';
+import { ArtifactElasticsearchProperties } from '@kbn/fleet-plugin/server/services';
+import { FoundExceptionListItemSchema } from '@kbn/securitysolution-io-ts-list-types';
+import { WebElementWrapper } from '../../../../../test/functional/services/lib/web_element_wrapper';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { InternalManifestSchemaResponseType } from './mocks';
+import { targetTags } from '../../target_tags';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'header']);
+  const queryBar = getService('queryBar');
+  const testSubjects = getService('testSubjects');
+  const endpointTestResources = getService('endpointTestResources');
+  const retry = getService('retry');
+  const esClient = getService('es');
+  const supertest = getService('supertest');
+  const find = getService('find');
+  const unzipPromisify = promisify(unzip);
+
+  describe('Endpoint Exceptions', function () {
```

Comment:
This test looks good, thanks for adding it. What do you think about adding these scenarios in `artifact_entries_list` test as we already check for the artifact results there?

Reply:
Good question - at first, I wanted to add these tests to `artifact_entries_list`, but then I realised that those tests (and all the helper functions) are built around artifacts under our management page, so it looked like an unnecessarily large effort to make the tests more universal to be reused for this case.

I think it works better this way - two smaller files with more encapsulated tests instead of one file with ~600 lines with two mostly independent test group.

Reply:
Sounds good, just in case it can be done, could we reuse som of the functions that check the `fleet-artifact` from the `artifact_entries_list` ?

Reply:
sure! I've extracted what looked like a common point
37795d7640b79fc2006049b175b1a0738ab0ffda